EasyProject/src/main/java/org/example/task1_case3/Matrix.java

```diff
@@ -0,0 +1,146 @@
+package org.example.task1_case3;
+import java.util.*;
+
+public class Matrix {
+
+    private final char[][] matrix;
+    private final int col;
+    private final int row;
+
+    public int getCol() {
+        return col;
+    }
+
+    public int getRow() {
+        return row;
+    }
+
+    public Matrix(char[][] m, int col ,int row) {
+        matrix= new char[col][row];
+        for (int i = 0; i < col; i++) {
+            System.arraycopy(m[i], 0, matrix[i], 0, row);
+
+        }
+        this.col=col;
+        this.row=row;
+    }
+
+    public Matrix(int col , int row){
+        matrix = new char[col][row];
+        this.col = col;
+        this.row = row;
+    }
+
+
+
+    public void playTicTacToe(){
+        inputCommand();
+    }
+    public void inputCommand(){
```

Comment:
1.does user have printed out choices? 
2. same here - use enum as option and not String -> (DOUBLE_USER("start user user")) ; (MIXED("start easy user")) ; etc

Reply:
1.done

Reply:
2.done

Reply:
don't know how to compare string from the enum with the one from the input